test/virtualization.test.ts

```diff
@@ -104,7 +104,7 @@ test('virtualization is enabled', async () => {
   assertRows(38, 3, 40);
 
   await scrollGrid({ scrollTop: 250 });
-  assertRows(39, 3, 41);
+  assertRows(38, 3, 40);
```

Comment:
We scrolled from 249 to 250 to ensure rendering stays consistent even when 1px changes.
If we render the same rows between these two points then we should find new positions 1px apart.

Reply:
Done